### DISCUSSION TITLE
Ansible : Getting my sql password connection error while running backend- main.yml / mysql- main.yml

### DISCUSSION THREAD

**Discussion by gkalluru77** [learndevopsonline/learndevopsonline#1584] [Q&A]
Getting below pasted error while running backend - main.yml file. I have used same code without creating directory structure, it was working fine that time.

backend-dev IP: 54.91.167.203
mysql-dev IP: 54.156.87.61

same error while running mysql- main.yml as well

TASK [backend : Load Schema] ****************************************************************************************************************************************************************
fatal: [backend-dev.gdevops.online]: FAILED! => {"changed": true, "cmd": "mysql -h mysql-dev.gdevops.online -uroot -pExpenseApp@1 < /app/schema/backend.sql", "delta": "0:00:00.023891", "end": "2023-11-08 15:35:56.190348", "msg": "non-zero return code", "rc": 1, "start": "2023-11-08 15:35:56.166457", "stderr": "mysql: [Warning] Using a password on the command line interface can be insecure.\nERROR 1045 (28000): Access denied for user 'root'@'ip-172-31-45-153.ec2.internal' (using password: YES)", "stderr_lines": ["mysql: [Warning] Using a password on the command line interface can be insecure.", "ERROR 1045 (28000): Access denied for user 'root'@'ip-172-31-45-153.ec2.internal' (using password: YES)"], "stdout": "", "stdout_lines": []}

**Comment by sreedharm07** [comment]
Hi @gkalluru77 can you ping your workstation ip

**Reply by gkalluru77** [reply]
workstation : 18.215.242.192

**Reply by gkalluru77** [reply]
shall stop the instancess?

**Reply by sreedharm07** [reply]
am checking alla the configeration was correct  @gkalluru77 am not getting where the error was 
can you please wait 10 mins

**Reply by gkalluru77** [reply]
or you want me to leave until you resolve the query

**Reply by gkalluru77** [reply]
Ok, will wait

**Reply by sreedharm07** [reply]
Hi @gkalluru77 
i clonned your repo into my machine and i runned same script in my local machine. its worked fine 
but in your machine its not working, that means we can assume your machine was overlapped with someother  processids or something i dont know much about this linux processes.
you just delete that instance and create again mysql instance and run, it will work 
the script is not working only in that machine. 
there is no error in your script thats why its taken time, sorry for taking time
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/cd5e8ebd-8721-4922-98ee-2f35242821f1)

**Reply by gkalluru77** [reply]
Ok i will try

**Reply by sreedharm07** [reply]
almost it will work but for conformation, please try and can you please update me here.

**Reply by gkalluru77** [reply]
It is working fine.
Thank you Sreedhar

**Reply by sreedharm07** [reply]
@gkalluru77 ok thank you for your time.

**Comment by gkalluru77** [comment]
Query resolved, hence closing the discusion
